static/src/javascripts/projects/common/modules/commercial/adblock-ask.js

```diff
@@ -25,35 +25,14 @@ const askHtml = `
 </div>
 `;
 
-export const adblockTest: ABTest = {
-    id: 'AdblockAsk',
-    start: '2019-02-20',
-    expiry: '2021-02-16',
-    author: 'Tom Forbes',
-    description:
-        'Places a contributions ask underneath the right-hand ad slot on articles.',
-    audience: 1,
-    audienceOffset: 0,
-    successMeasure: '',
-    audienceCriteria: '',
-    showForSensitive: true,
-    canRun() {
-        return (
-            !shouldHideSupportMessaging() &&
-            !pageShouldHideReaderRevenue() &&
-            !config.get('page.hasShowcaseMainElement')
-        );
-    },
+const canShow = () =>
+    !shouldHideSupportMessaging() &&
+    !pageShouldHideReaderRevenue() &&
+    !config.get('page.hasShowcaseMainElement');
 
-    variants: [
-        {
-            id: 'control',
-            test: (): void => {
-                const slot = document.querySelector('.js-aside-slot-container');
-                if (slot) {
-                    slot.innerHTML += askHtml;
-                }
-            },
-        },
-    ],
+export const initAdblockAsk = () => {
+    const slot = document.querySelector('.js-aside-slot-container');
```

Comment:
This seems like a good candidate for using [fastdom-promise](https://github.com/wilsonpage/fastdom) as  we do elsewhere in frontend.  This library "Eliminates layout thrashing by batching DOM read/write operations" , we could use `fastdom.read` to get the `slot` and `fastdom.write` to set the `innerHTML`

Reply:
Thanks George - have done 👍